Source/Classes/AnimatedImages/PINAnimatedImage.m

```diff
@@ -58,13 +76,13 @@ - (CFTimeInterval)totalDuration
 
 - (NSUInteger)frameInterval
 {
-    return MAX(self.minimumFrameInterval * kPINAnimatedImageDisplayRefreshRate, 1);
+    return MAX(self.minimumFrameInterval * [PINAnimatedImage maximumFramesPerSecond], 1);
 }
 
 //Credit to FLAnimatedImage ( https://github.com/Flipboard/FLAnimatedImage ) for display link interval calculations
 - (NSTimeInterval)minimumFrameInterval
 {
-    const NSTimeInterval kGreatestCommonDivisorPrecision = 2.0 / kPINAnimatedImageMinimumDuration;
+    const NSTimeInterval kGreatestCommonDivisorPrecision = 2.0 / (1.0 / [PINAnimatedImage maximumFramesPerSecond]);
```

Comment:
Just a though, since animated GIF playback is performance sensitive perhaps this value should also be `dispatch_once`'d. Prior to this the value could be computed at compile time and was likely just a constant while running, now since it's dynamic is has to do this division on every single pass. Could skip work by computing it one time, since `maximumFramesPerSecond` is only ever computed one time.

Reply:
Good point.